backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -482,6 +486,42 @@ export class AdminResolver {
       maxPerCycle,
     }: ContributionLinkArgs,
   ): Promise<ContributionLink> {
+    logger.debug(
+      `createContributionLink(validFrom=${validFrom}, validTo=${validTo}, name=${name}, amount=${amount}, memo=${memo}, cycle=${cycle}, maxPerCycle=${maxPerCycle}, maxAmountPerMonth=${maxAmountPerMonth})...`,
+    )
+    if (!isStartEndDateValid(validFrom, validTo)) {
+      logger.error(`The validFrom=${validFrom} must be before or equals the validTo=${validTo}!`)
+      throw new Error(`The validFrom=${validFrom} must be before or equals the validTo=${validTo}!`)
+    }
+    if (name == null) {
+      logger.error(`The name must be initialized!`)
+      throw new Error(`The name must be initialized!`)
+    }
+    if (
+      name.length < CONTRIBUTIONLINK_NAME_MIN_CHARS ||
+      name.length > CONTRIBUTIONLINK_NAME_MAX_CHARS
+    ) {
+      const msg = `The name=${name} with a length of ${name.length} did not fulfill the requested bounderies min=${CONTRIBUTIONLINK_NAME_MIN_CHARS} and max=${CONTRIBUTIONLINK_NAME_MAX_CHARS}`
+      logger.error(`${msg}`)
+      throw new Error(`${msg}`)
+    }
+    if (
+      memo.length < CONTRIBUTIONLINK_MEMO_MIN_CHARS ||
+      memo.length > CONTRIBUTIONLINK_MEMO_MAX_CHARS
+    ) {
+      const msg = `The memo=${memo} with a length of ${memo.length} did not fulfill the requested bounderies min=${CONTRIBUTIONLINK_MEMO_MIN_CHARS} and max=${CONTRIBUTIONLINK_MEMO_MAX_CHARS}`
+      logger.error(`${msg}`)
+      throw new Error(`${msg}`)
+    }
+    if (amount == null) {
+      logger.error(`The amount must be initialized!`)
+      throw new Error('The amount must be initialized!')
+    }
+    const amountObj = new Decimal(amount)
+    if (amountObj.lessThanOrEqualTo(new Decimal(0))) {
```

Comment:
I would test not greater than 0. This one looks like as 0 would pass.

Reply:
is replaced by "if (!new Decimal(amount).isPositive())"